Javascript/homework/responses/2.1-css-linking.md

```diff
@@ -0,0 +1,12 @@
+### Week 2 - Designing a digital clock using CSS
+
+*In this week's assignment, students will build off of last weeks work and style their existing webpage to produce a more aesthetically pleasing digital clock.*
+
+#### Task 1: Linking your CSS file
+
+This week, we're going to take the HTML and JavaScript code we wrote last week and style our page using CSS to make it more aesthetically pleasing.  First, copy your index.html file into a new html file in your week 2 homework folder.
+
+To style our webpage, we must link a css file to our HTML file.  To do this, add a `link` tag inside the head of your html file. This tag should have the `rel` attribute set to `"stylesheet"` and the `href` attribute set to `"style.css"`.
+
+Next, open a new file in your week 2 folder and save it as `style.css`. It's important that your file name matches the one you typed in your HTML document in its link tag's `href` attribute, as this will be what links your two files.
+
```

Comment:
I personally see no issue. And if someone doesn't know how to write the code properly maybe we can include it. But it's simply googling how to href a file.